backend/src/helpers/token/create-token/create-token.ts

```diff
@@ -0,0 +1,9 @@
+import { ENV } from '~/common/enums/enums';
+import jwt, { JwtPayload, Secret, SignOptions } from 'jsonwebtoken';
+
+export const createToken = (data: JwtPayload): string =>
+  jwt.sign(
+    data,
+    ENV.JWT.SECRET as Secret,
+    { expiresIn: ENV.JWT.EXPIRES_IN } as SignOptions,
+  );
```

Comment:
```suggestion
const createToken = (data: JwtPayload): string => {
  return jwt.sign(
    data,
    ENV.JWT.SECRET as Secret,
    { expiresIn: ENV.JWT.EXPIRES_IN } as SignOptions,
  );
}

export { createToken }
```
We are using export at the bottom of the file and don't forget to always use curly braces (if it's not one-liner)